content/blog/building-original-projects/index.md

```diff
@@ -0,0 +1,285 @@
+---
+{
+  title: "How to Build Original Projects",
+  description: "People often say that you should build your own projects to grow as a developer, without explaining how. Let's learn how to actually plan and build your own custom projects.",
+  published: '2023-10-31',
+  authors: ["reyes-dev"],
+  tags: ["opinion"],
+  license: 'cc-by-4'
+}
+---
+
+There's a lot of different ways to practice programming. You can solve coding exercises, follow tutorials and courses that walk you through the process of learning syntax and concepts, or read books on a particular programming language. Maybe you've done some or all of these, but don't feel enthusiastic about your results. You're not growing or having much fun, yet you feel obligated to keep consuming material in order to obtain skills.
+
+I'm going to suggest another way to grow as a software developer that you might want to experiment with: building original projects. The broadest definition of an original project can be boiled down to:
+- It is a new creation of your mind; there is no guide or tutorial that can teach you directly how to build it
+- It solves a real world problem you or others have
+
+To clarify, an original project is not necessarily a completely new product that nobody has ever thought of, or invented before (although it can be!).  I also don't mean that you are *not allowed* to reference guides, tutorials, documentation, other people's code or Google in the process of building an original project. The opposite is true, and I highly encourage using all resources at your disposal.
+
+Rather, the essence of the original project is autonomy and personal vision. You are laying all the groundwork: the ideas, plan, research, and features. Then, you will execute on your plan of action by translating the broad ideas into code. By the end, you will have in your hands new software for your use case. You will be trading in the prepackaged, tutorial-project approach for a slightly scarier, but ultimately more rewarding approach, driven mainly by your creativity and problem-solving skills.
```

Comment:
Guidance for future blog posts - smaller paragraphs. We've found that - at least with technical writing - they're:

- Easier to read at a glance
- Enhance user retention in reading

Try to stay to 2-3 sentences (~300-350 characters) at most.